docker-compose.yml

```diff
@@ -11,4 +11,4 @@ services:
             max-size: 10m
             max-file: "3"
         ports:
-          - '5432:5432'
\ No newline at end of file
+          - '25:25'
```

Comment:
I think this line is incorrect. Postgres in the container is listening on 5432 (the standard postgres port). If you which to expose this on the host machine as a different port you want something like "25:5432" eg make port 5432 public on the host machine as port 25